From f36b3abe41179163a6d556cf727ea3942daab115 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Fri, 29 Sep 2017 10:20:39 +1000
Subject: [PATCH 3974/5855] amdgpu/dc: drop dc_ver char

This isn't referenced anywhere, and if it was it should be const.

Signed-off-by: Dave Airlie <airlied@redhat.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index 3d47b03..160132e 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -49,8 +49,6 @@
 #include "mem_input.h"
 
 
-char dc_ver[] = DC_VER;
-
 /*******************************************************************************
  * Private functions
  ******************************************************************************/
-- 
2.7.4

